src/flights/url.py

```diff
@@ -0,0 +1,48 @@
+from fast_flights import FlightData, Passengers, create_filter
+
+def generate_flight_url(
+        departure_date,
+        from_airport,
+        to_airport,
+        trip_type="one-way",
+        seat_type="economy",
+        adults=1,
+        children=0,
+        infants_in_seat=0,
+        infants_on_lap=0,
+        currency="EUR"
+):
+    
+    filter = create_filter(
+        flight_data=[
+            FlightData(
+                date = departure_date,
+                from_airport=from_airport,
+                to_airport=to_airport,
+            )
+        ],
+        trip=trip_type,
+        seat=seat_type,
+        passengers=Passengers(
+                adults=adults,
+                children=children,
+                infants_in_seat=infants_in_seat,
+                infants_on_lap=infants_on_lap,
+        ),
+    )
+
+    b64=filter.as_b64().decode('utf-8')
+    return f"https://www.google.com/travel/flights?tfs={b64}&curr={currency}"
+
+#Exemple
+
```

Comment:
put example in a main function:
```
def main():
   ...
```

and then define, at the bottom of the file:

```
if __name__ == "__main__":
   main()
```